stable/datadog/templates/system-probe-configmap.yaml

```diff
@@ -0,0 +1,358 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: system-probe-config
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app: {{ $.Chart.Name }}
+    service: {{ $.Chart.Name }}
+    chart_version: {{ $.Chart.Version }}
```

Comment:
should we include the labels from https://github.com/helm/charts/pull/14674/files#diff-c5ed47d16bb168ac09be3d6cdf6b4abaL11 ?